db/schema.rb

```diff
@@ -44,6 +54,7 @@
     t.string   "last_sign_in_ip"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.string   "name"
```

Comment:
Yeah, that does seem better. I also want to do some real migration practice, so this could prove useful.